app/src/main/java/com/glia/exampleapp/Utils.java

```diff
@@ -230,4 +231,24 @@ public static Integer getTypefaceFromPrefs(SharedPreferences sharedPreferences,
     public static boolean getUseOverlay(SharedPreferences sharedPreferences, Resources resources) {
         return sharedPreferences.getBoolean(resources.getString(R.string.pref_use_overlay), true);
     }
+
+    public static ScreenSharing.Mode getScreenSharingModeFromPrefs(
+            SharedPreferences sharedPreferences,
+            Resources resources
+    ) {
+        String value = sharedPreferences.getString(
+                resources.getString(R.string.pref_screen_sharing_mode),
+                resources.getString(R.string.default_value)
+        );
+        String unbounded = resources.getString(R.string.screen_sharing_mode_unbounded);
+        String appBounded = resources.getString(R.string.screen_sharing_mode_app_bounded);
+
+        if (value.equals(unbounded)) {
+            return ScreenSharing.Mode.UNBOUNDED;
+        } else if (value.equals(appBounded)) {
+            return ScreenSharing.Mode.APP_BOUNDED;
+        } else {
+            return null;
```

Comment:
else here seems unnecessary - return default

Reply:
why not use toggle instead?

Reply:
I assumed we need a default value as for some of the other settings. But if this is not needed, sure toggle is better, and the default value can be set somewhere deeper in the SDK.